content/service-design-delivery-process/alpha-stage/test.md

```diff
@@ -13,7 +13,7 @@ Test with users using paper prototypes first. As you learn from your user resear
 A good prototype is not the real service but needs to show how the service works. The prototype must:
 - look and feel like a real digital service
 - have enough features for users to interact with
-- provide a seamless user experience along the ‘happy path’
+- show a seamless user experience along the ‘happy path’
```

Comment:
I always wondered what `happy path` meant. Would be good to fix this separately.